articles/azure/azs-how-create-spn-portal.md

```diff
@@ -0,0 +1,112 @@
+
+## Prerequisites
+
+- An active Azure Subscription
+
+## Create an SPN account for Azure and Azure Stack
+
+### Create an SPN
+
+1. Log in to the [Azure portal](https://portal.azure.com).
+
+2. In the favourites panel, select **Azure Active Directory**.
+
+    ![Select Azure Active Directory](images/azs-browser-select-aad.png)
+
+3. In the *Azure Active Directory* blade, select **App registrations**.
+
+    ![Select App registrations](images/azs-browser-app-registrations.png)
+
+4. On the *App registrations* page, click the **+ New application registration** button.
+
+5. In the *Create* blade, enter the following information:
+
+   - **Name** - The name of the application. Should have a minimum of 4 characters.
+
+   - **Application Type** - Choose "Web app / API" to add an application that represents a web application, a web API, or both. Choose "Native" to add an application that can be installed on a user's device or computer. In this instance, you can use `Web app / API`.
+
+   - **Sign-on URL** - The URL where user can sign in and use your app. You can change this URL later. In this instance, you can use `http://localhost`.
+
+    ![Create Service Principal](images/azs-browser-create-spn.png)
+
+6. Click **Create**.
+
+### Gathering SPN Credentials
+
+1. On the blade that has now opened, copy the `Application ID` and store it somewhere secure. This value is sometimes referred to as the SPN Client ID.
+
+    ![Copy Application ID](images/azs-browser-select-SPN-App-ID.png)
+
+2. On the same blade, click the **Settings** button.
+
+3. In the *API Access* section of the *Settings* blade, select **Keys**.
+
+4. In the *Keys* blade, enter a description for a key as well as an expiry duration, then click **Save**.
+
+    ![Create SPN key](images/azs-browser-add-SPN-key.png)
+
+5. Copy the displayed value for the key and store it somewhere secure. This value is referred to as the SPN Client Secret.
+
+### Configure role-based access control for the SPN on Azure Stack
+
+1. Log in to the UKCloud Azure Stack portal.
+
+    For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md).
+
+2. Navigate to the resource you wish to give the SPN access to. In this example we are adding the SPN to the subscription resource to give it permission to all resources.
+
+   1. View your subscription by clicking **All services** in the favourites panel, then selecting **Subscriptions** under the *General* section.
+
+   2. Select your subscription.
+
+   3. On your subscription's blade, select **Access Control (IAM)**.
+
+3. On the *Access Control (IAM)* blade, click the **+ Add** button.
+
+    ![Add SPN to RBAC](images/azs-browser-add-RBAC-user.png)
+
+4. Select the role you wish to add your SPN to.
+
+5. Enter the name of the SPN in the *Select* field, select the SPN when it appears then click **Save**
+
+    ![Add SPN to role](images/azs-browser-add-SPN-role.png)
```

Comment:
image includes a lot of white space. Not sure if that's because its a weird dialog box, or if you've removed some sensitive data. Would suggest removing the white space.

Reply:
The UI actually has this unnecessary amount of white-space...  I've trimmed the image down so it doesn't look awful

Reply:
Looks much better. May not exactly match the UI, but user will be able to see they are in the right place.